bob/src/cluster/operations.rs

```diff
@@ -295,7 +276,48 @@ pub(crate) async fn put_local_node(
     backend.put_local(key, data, op).await
 }
 
+// =================== EXIST ==================
 
+pub(crate) async fn exist_on_local_node(
+    backend: &Backend,
+    keys: &[BobKey],
+) -> Result<Vec<bool>, Error> {
+    Ok(backend
+        .exist(keys, &BobGetOptions::new_get(Some(GetOptions::new_local())))
+        .await?)
+}
+
+pub(crate) async fn exist_on_local_alien(
+    backend: &Backend,
+    keys: &[BobKey],
+) -> Result<Vec<bool>, Error> {
+    Ok(backend
+        .exist(keys, &BobGetOptions::new_get(Some(GetOptions::new_alien())))
+        .await?)
+}
+
+pub(crate) async fn exist_on_remote_nodes(
+    nodes: &[Node],
+    keys_by_node: HashMap<Node, Vec<BobKey>>,
+) -> Vec<Result<NodeOutput<Vec<bool>>, NodeOutput<Error>>> {
+    LinkManager::call_nodes(nodes.iter(), |client| {
```

Comment:
Looks like you can use `keys_by_node.keys()` as the first parameter. In this way you don't need to pass `nodes` slice as a separate parameter